content/millennium/r4/clinical/care-provision/nutrition-order.md

```diff
@@ -123,6 +128,17 @@ List an individual NutritionOrder by its id:
 
 <%= disclaimer %>
 
+#### Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/NutritionOrder/313790117
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_nutrition_order_child_entry) %>
+
+<%= disclaimer %>
+
```

Comment:
I was thinking we could just include the child order as an example in the search output (just another returned Order). We probably don't need to do another request and response block here.

Reply:
@joeratt - I may be missing the context of your comment but for search by patient we don't return the child orders. We only return them on search by id and read by id.

Reply:
[commit](https://github.com/cerner/fhir.cerner.com/pull/679/commits/1d4fec2c38cd8feb2c5dfb26f498423ebbffca2c#diff-63c407fea7510167107b85f2810bc5775ff46200c2554d092fcb3df7a3ea2aaeL131) Can change back if this is found to be wrong.

Reply:
[commit](https://github.com/cerner/fhir.cerner.com/pull/679/commits/19e5f600184edee33d8d2ce15a5fdbcd141dc957#diff-63c407fea7510167107b85f2810bc5775ff46200c2554d092fcb3df7a3ea2aaeR131) undoing changes

Reply:
looks good